xbmc/windowing/WinSystem.cpp

```diff
@@ -283,14 +283,19 @@ void CWinSystemBase::UnregisterRenderLoop(IRenderLoop *client)
 
 void CWinSystemBase::DriveRenderLoop()
 {
-  m_winEvents->MessagePump();
+  MessagePump();
 
   { CSingleLock lock(m_renderLoopSection);
     for (auto i = m_renderLoopClients.begin(); i != m_renderLoopClients.end(); ++i)
       (*i)->FrameMove();
   }
 }
 
+bool CWinSystemBase::MessagePump()
+{
+  return m_winEvents->MessagePump();
```

Comment:
This should return false. Subclasses override methods to add behaviour not to reduce it. Winsystems that  have winevents should override it